cime_config/testdefs/testmods_dirs/clm/FatesColdSeedDisp/README

```diff
@@ -0,0 +1,23 @@
+Testing FATES cross grid seed dispersal is activated by enabling the 
+namelist switch option fates_seeddisp_cadence as well as providing reasonable
+values to the fates parameter file for the following variables:
+
+fates_seed_dispersal_fraction
+fates_seed_dispersal_max_dist
+fates_seed_dispersal_pdf_scale
+fates_seed_dispersal_pdf_shape
+
+Given that the default fates parameter file has the above variables as unset,
+a custom fates parameter file must be supplied to appropriately test this mode.
+This testmod itself addresses CTSM issue #2151.
```

Comment:
Here I think you should maybe just say the content of #2151 rather than reference a number that doesn't link to github. In the issue we should start marking what parts of it are addressed with this one.

Reply:
Per our discussion, I'll add the url linking to the issue.